package.json

```
@@ -1,22 +1,28 @@
 {
-  "name": "zjs",
+  "name": "zwave-js-server",
   "version": "1.0.0",
-  "description": "",
-  "main": "build/index.js",
+  "description": "Full access to zwave-js driver throught Websockets",
+  "main": "dist/lib/index.js",
+  "bin": {
+    "zwave-server": "dist/bin/server.js",
+    "zwave-client": "dist/bin/client.js"
+  },
   "scripts": {
     "lint": "eslint --ext .ts .",
-    "lint:fix": "eslint --fix --ext .ts ."
+    "lint:fix": "eslint --fix --ext .ts .",
+    "build": "tsc -p ."
```

Just to make sure we always have the latest build included with release:

```suggestion
    "build": "tsc -p .",
    "prepublishOnly": "rm -rf dist && npm run build"
```